scripts/run.py

```diff
@@ -240,7 +242,7 @@ def do_import(subdir, srcpath, destpath):
 
 
 @task
-def compile(project_path, backwards_steps=BASE_CONFIG["max_backwards_steps"]):
+def compile(project_path, backwards_steps=BASE_CONFIG["max_backwards_steps"], pull=True):
```

Comment:
Are we keeping this? Not sure what it tries to do.

Reply:
Good point, we should remove it. Sorry